doc/lava_usecase.rst

```diff
@@ -0,0 +1,95 @@
+===============================
+Use case: setup SQUAD with LAVA
+===============================
+
+Introduction
+------------
+
+Once SQUAD installation is complete, a typical use case would be integrating
+it with a LAVA instance. The purpose of this use case is to describe a
+step-by-step set up of SQUAD with LAVA to submit and fetch jobs. If you haven't
+yet set up SQUAD, take a step back and follow :ref:`production_install_ref_label`
+
+
+Setting up a LAVA instance
+--------------------------
+
+LAVA has its own extensive documentation on how to get a server up and running.
+If you don't have it already, please refer to `LAVA installation`_ to configure
+yourself one. From this point it's taken that you have enough access to a
+running LAVA v2 instance that jobs can be submitted to and fetched from.
+
+.. _`LAVA installation`: https://validation.linaro.org/static/docs/v2/installing_on_debian.html#debian-installation
+.. _`LAVA authentication tokens`: https://validation.linaro.org/static/docs/v2/first_steps.html?highlight=token#authentication-tokens
+
```

Comment:
I think we should mention squad uses LAVA's 'event notification' feature which is disabled by default. User should enable it in their LAVA instance. 

Refer to https://validation.linaro.org/static/docs/v2/data-export.html?highlight=event%20notifications#event-notifications

Here is an example https://github.com/Linaro/lkft-remote-lab/blob/master/LAVA/deploy-distributed-lava-instance.md#lava-master #4

Reply:
Thanks for the input @chase-qi ! I've included a note mentioning that event notifications should be enabled and pointed the user to this URL you sent.